cas/ctg_store.c

```diff
@@ -1001,7 +1130,17 @@ static int ctg_op_tick_ret(struct m0_ctg_op *ctg_op,
 	return ret;
 }
 
-static int ctg_op_exec(struct m0_ctg_op *ctg_op, int next_phase)
+/*
+ * Returns a bitmap of zones that could be used within the given operation.
+ * See ::m0_be_alloc_zone_type.
+ */
+static uint32_t ctg_op_zones(const struct m0_ctg_op *ctg_op)
+{
+	return M0_BITS(M0_BAP_NORMAL) |
+		((ctg_op->co_flags & COF_RESERVE) ? M0_BITS(M0_BAP_REPAIR) : 0);
+}
+
+static int ctg_op_exec_normal(struct m0_ctg_op *ctg_op, int next_phase)
```

Comment:
Is normal an opposite to versioned? May be it's more readable to use _nonversioned_ in this case?

Reply:
I think *normal* is OK here.

Reply:
I agree with @huanghua78 .

The important property of normal operations is that they are the baseline. It is not just "non-versioned". It is "regular, usual, familiar CAS". I think eventually there will be no different code paths for "normal" and "versioned" behavior. But until that moment, let's call the CAS that we know as "normal".